internal/service/dns_config/api_auth_zone_data_source.go

```diff
@@ -90,6 +89,9 @@ func (d *AuthZoneDataSource) Configure(ctx context.Context, req datasource.Confi
 
 func (d *AuthZoneDataSource) Read(ctx context.Context, req datasource.ReadRequest, resp *datasource.ReadResponse) {
 	var data ConfigAuthZoneModelWithFilter
+	var offset int32 = 0
+	var limit int32 = 1000
```

Comment:
can we make the limit a common constant?

Reply:
fixed

Reply:
It's a const but within the function. I would have preferred it to be an accessible constant

Reply:
Resolved